eq-author/src/components/CollapsibleNavItem/index.js

```diff
@@ -60,25 +59,36 @@ const Title = styled.button`
   &:disabled {
     background: ${colors.orange};
     outline: none;
-    color: ${colors.black};
 
-    svg {
+    > span {
+      color: ${colors.black};
+    }
```

Comment:
```suggestion
    ${Title} {
      color: ${colors.black};
    }
```
Just need to move title above the button then